lib/bemhtml/index.js

```diff
@@ -180,7 +180,16 @@ BEMHTML.prototype.renderClose = function renderClose(prefix,
     if (content || content === 0)
       out += this.renderContent(content, isBEM);
 
-    out += '</' + tag + '>';
+    var optionalClosingTags = [
+      // html4
+      'html', 'head', 'body', 'p', 'ul', 'ol', 'li', 'dt', 'dd',
+      'colgroup', 'thead', 'tbody', 'tfoot', 'tr', 'th', 'td', 'option',
+      // html5
+      /* dl — Neither tag is omissible */ 'rb', 'rt', 'rtc', 'rp', 'optgroup'
```

Comment:
Please add comment with links to specification: 
1) https://www.w3.org/TR/html5/syntax.html#optional-tags
2) https://html.spec.whatwg.org/multipage/syntax.html#optional-tags